suricata.yaml.in

```diff
@@ -717,6 +718,8 @@ app-layer:
       enabled: yes
       detection-ports:
         dp: 5900, 5901, 5902, 5903, 5904, 5905, 5906, 5907, 5908, 5909
+    mqtt:
+      enabled: yes
```

Comment:
Do we want to enable it by default right away ?

Reply:
I'm fine with both. What do the others think?

Reply:
Disabled it for now by default, since it is so easy to turn back on.